modules/nextMillenniumBidAdapter.js

```diff
@@ -26,6 +46,8 @@ export const spec = {
     _each(validBidRequests, function(bid) {
       window.nmmRefreshCounts[bid.adUnitCode] = window.nmmRefreshCounts[bid.adUnitCode] || 0;
       const id = getPlacementId(bid)
+      const auctionId = bid.auctionId
+      const bidId = bid.bidId
       let sizes = bid.sizes
```

Comment:
`sizes` is always referencing `banner` sizes even if the video player is a different size.

Reply:
@lksharma You're right. If you look at postbody you find param ext.prebid.storedrequest. In our side all sizes are predefined. So sizes that are passed via banner size, are used by our server in special cases

Reply:
Is it possible for your servers to return a video ad for a banner-only ad unit (example: 300x250)?  My concern here is that for a purely banner ad unit you might return a video response. In that case, the ad will not render since the video ad unit requires a `renderer` to be specified (for outstream) or a cache URL to be defined  (for instream).

Reply:
> Is it possible for your servers to return a video ad for a banner-only ad unit (example: 300x250)? My concern here is that for a purely banner ad unit and the backend returns a video response then the ad will not render. This is because video media types require a `renderer` to be specified (for outstream) or a cache URL to be defined (for instream).
@lksharma 
No. It is not possible. Because every placement ID is configured for specific publisher and placement. So if placement_id is configured for video, it will be used only for video player. The same case is for banner. As I said before, banner sizes we send only for our internal logic which does not break anything on the client side.